qcodes/tests/drivers/test_ami430.py

```diff
@@ -193,7 +193,11 @@ def test_instantiation_from_badly_typed_argument(
 
 
 @given(set_target=random_coordinates["cartesian"])
-@settings(max_examples=10, suppress_health_check=(HealthCheck.function_scoped_fixture,))
+@settings(
+    max_examples=10,
+    suppress_health_check=(HealthCheck.function_scoped_fixture,),
+    deadline=None,
```

Comment:
do we need to also drop the deadlines in the ami430 VISA tests? or are those "fine"?